### DISCUSSION TITLE
add show() as an alias for df() just for compatibility reason with pyspark

### DISCUSSION THREAD

**Discussion by djouallah** [duckdb/duckdb#6218] [Ideas]
I have a function that print query results from a sql statement, in Pyspark we can do something like

spark.sql(sql).show() and it will print result 

I guess in duckdb , we can do duckdb.sql().df() can we have an alias just for convenience :)

**Comment by Mytherin** [comment]
I would say a method printing the result would be more appropriate then, no?

**Comment by djouallah** [comment]
@Mytherin yes :)
